### PR TITLE
feature(REPORT): Bold reports V5.4 Release Changes

### PR DESCRIPTION
**Pull request by Vishnu7101**
<table>
    <tr>
        <td>Feature description</td>
        <td>
To Update the Bold Reports -angular-reporting-components public repo with the  Bold-Reports 5.4 Release Changes</td>
    </tr>
    <tr>
        <td>Analysis</td>
        <td> Analyzed about Bold reports V5.4 release changes</td>
    </tr>
    <tr>
        <td>Reason for not identifying earlier</td>
        <td>
            It's a new feature</td>
    </tr>
    <tr>
        <td>Is it a breaking issue?</td>
        <td>No, It's a new feature </td>
    </tr>
    <tr>
        <td>Solution description</td>
        <td>
<li> Update the version in package json to 5.4.20 in relevant to Bold-Reports 5.4 Release Changes
<li>To modify the files according to the latest changes made in Bold report V5.4
</td>
    </tr>
    <tr>
        <td>Areas affected and ensured</td>
        <td>NA</td>
    </tr>
    <tr>
        <td>Test cases</td>
        <td>NA</td>
    </tr>
    <tr>
        <td>Test bed sample locations</td>
        <td>VS & VS Code</td>
    </tr>
</table>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ## Bold Reports Angular UI Components
 
-The `Bold Reports` UI components contains [Angular Report Viewer](https://www.boldreports.com/embedded-reporting/angular-report-viewer?utm_source=github&utm_medium=backlinks) and [Angular Report Designer](https://www.boldreports.com/embedded-reporting/angular-report-designer?utm_source=github&utm_medium=backlinks) reporting components for enterprise web development. Empower your web application with feature-rich report preview, edit, and customization capabilities that allow you to explore data easily and make real-time decisions. Effortlessly print and export reports.
+The `Bold Reports` UI components contains [Angular Report Viewer](https://www.boldreports.com/embedded-reporting/angular-report-viewer) and [Angular Report Designer](https://www.boldreports.com/embedded-reporting/angular-report-designer) reporting components for enterprise web development. Empower your web application with feature-rich report preview, edit, and customization capabilities that allow you to explore data easily and make real-time decisions. Effortlessly print and export reports.
```
Comment (karthickthangasamy) [inline README.md:3]: please revert these changes.

**Review by karthickthangasamy — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Vishnu7101) [reply README.md]: Updated

**Review by karthickthangasamy — APPROVED** [review]
